manim/mobject/shape_matchers.py

```diff
@@ -117,13 +119,44 @@ def get_fill_color(self):
 
 
 class Cross(VGroup):
-    def __init__(self, mobject, stroke_color=RED, stroke_width=6, **kwargs):
-        VGroup.__init__(
-            self,
-            Line(UP + LEFT, DOWN + RIGHT),
-            Line(UP + RIGHT, DOWN + LEFT),
+    """Creates a cross.
+
+    Parameters
+    ----------
+    mobject
+        The mobject linked to this instance. It fits the mobject when specified. Defaults to None.
+    stroke_color
+        Specifies the color of the cross lines. Defaults to Red.
+    stroke_width
+        Specifies the width of the cross lines. Defaults to 6.
+    scale_factor
+        Scales the cross to the provided units. Defaults to 1.
+
+    Examples
+    --------
+    .. manim:: ExampleCross
+        :save_last_frame:
+
+        class ExampleCross(Scene):
+            def construct(self):
+                cross = Cross()
+                self.add(cross)
+    """
+
+    def __init__(
+        self,
+        mobject: Optional[Mobject] = None,
+        stroke_color: Optional[Color] = RED,
+        stroke_width: Optional[float] = 6,
+        scale_factor: Optional[float] = 1,
```

Comment:
`Optional` is for when `None` is also a valid paramter, in these cases, it's not. So Optional is not appropriate here.
```suggestion
        stroke_color: Color = RED,
        stroke_width: float = 6,
        scale_factor: float = 1,
```